gcloud-java-contrib/gcloud-java-nio-examples/src/main/java/com/google/gcloud/nio/examples/ListFilesystems.java

```diff
@@ -43,4 +43,5 @@ private static void listFilesystems() {
       System.out.println("  " + p.getScheme());
     }
   }
+
 }
```

Comment:
@aozarov is right. I meant to add a newline before the EOF and remove the one before the last `}`. To give some context here [why should text files end with a newline?](http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline).